### PR TITLE
add a model_configure template for global nest

### PR DESCRIPTION
**Pull request by guoqing-noaa**
## Commit Queue Requirements:
- [x] Fill out all sections of this template.
- [x] All sub component pull requests have been reviewed by their code managers.
- [ ] Run the full Intel+GNU RT suite (compared to current baselines) on either Hera/Derecho/Hercules
- [ ] Commit 'test_changes.list' from previous step
---
## Description:
add a model_configure template for global nest

### Commit Message:
add a model_configure template for global nest
`tests/parm/model_configure_nest.IN`

### Priority:
* High: Need this template file to run the global-nest workflow 

### UFSWM:

* None

### Sub component Pull Requests:
* None

### UFSWM Blocking Dependencies:
* None

---
### Regression Test Changes (Please commit test_changes.list):
* No Baseline Changes.
* Just add a model_configure template under tests/parm, no need to run RT suite and commit test_changes.list.

### Input data Changes:

* None.

### Library Changes/Upgrades:
* No Updates
  
---

## Testing Log:
- RDHPCS
  - [ ] Hera
  - [ ] Orion
  - [ ] Hercules
  - [ ] Jet
  - [ ] Gaea
  - [ ] Derecho
- WCOSS2
  - [ ] Dogwood/Cactus
  - [ ] Acorn
- [ ] CI
- [ ] opnReqTest (complete task if unnecessary)

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
@jkbk2004 This PR is requesting high priority.

**Comment by DusanJovic-NOAA** [comment]
Which regression test is using (testing) this template?

**Comment by guoqing-noaa** [comment]
> Which regression test is using (testing) this template?

This is a new template for the global nest applications. No existing regression tests for it.
@SamuelTrahanNOAA is trying to add a global nest regression test to this repo. Could you convert this PR to a draft PR at this moment? Thanks!

**Comment by SamuelTrahanNOAA** [comment]
There are regression tests in the repository already. I'm replacing one of them.

These are the tests:

1. `gnv1_c96_no_nest_debug` - unnested uncoupled C96 GFS with global_nest_v1 physics suite. Code is compiled in debug mode. Run for GNU and Intel compilers.  It uses the exact same C96 input as dozens of other ufs-weather-model regression tests.
2. `gnv1_nested` - HAFS North Atlantic global nested test case with global_nest_v1 physics. This is the same input used for other HAFS global cases. Compiled optimized; intel only.

I'm replacing the second one (`gnv1_nested`). It'll use the Water in the West domain instead of the HAFS North Atlantic domain. Also, it'll be based on the GFS-like regression tests (using control_run.IN) so it'll be more compatible with the global-workflow.

File: tests/parm/model_configure_nest.IN [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+jmo:                     @[NEST_JMO]
+cen_lon:                 @[TARGET_LON]
+cen_lat:                 @[TARGET_LAT]
+lon1:                    @[NEST_LON1]
+lat1:                    @[NEST_LAT1]
+lon2:                    @[NEST_LON2]
+lat2:                    @[NEST_LAT2]
+dlon:                    @[NEST_DLON]
+dlat:                    @[NEST_DLAT]
+</output_grid_02>
```
Comment (SamuelTrahanNOAA) [inline tests/parm/model_configure_nest.IN:53]: These are the variables currently used in the regression tests:

```suggestion
<output_grid_02>
output_grid:             @[OUTPUT_GRID_2]
imo:                     @[IMO_2]
jmo:                     @[JMO_2]
cen_lon:                 @[CEN_LON_2]
cen_lat:                 @[CEN_LAT_2]
lon1:                    @[LON1_2]
lat1:                    @[LAT1_2]
lon2:                    @[LON2_2]
lat2:                    @[LAT2_2]
dlon:                    @[DLON_2]
dlat:                    @[DLAT_2]
</output_grid_02>
```

**Review by SamuelTrahanNOAA — CHANGES_REQUESTED** [review]

**Comment by SamuelTrahanNOAA** [comment]
I'm unclear on what is gained by merging this PR. The reason it is high priority is that a global-workflow PR needs a model_configure for the global_nest_v1 suite. However, the global-workflow PR generates an incorrect input.nml. Its namelist settings don't match the ccpp suite. The new file isn't used by any regression test yet.

Also, the `NEST_` variables won't work long-term. What if there is more than one nest?

I'll have another PR soon with a regression test that has a correct namelist.

**Comment by guoqing-noaa** [comment]
> I'm unclear on what is gained by merging this PR. The reason it is high priority is that a global-workflow PR needs a model_configure for the global_nest_v1 suite. However, the global-workflow PR generates an incorrect input.nml. Its namelist settings don't match the ccpp suite. The new file isn't used by any regression test yet.
> 
> Also, the `NEST_` variables won't work long-term. What if there is more than one nest?
> 
> I'll have another PR soon with a regression test that has a correct namelist.

@SamuelTrahanNOAA If you will create another PR,  I will close this PR. Thanks!

**Comment by SamuelTrahanNOAA** [comment]
My replacement PR is #2284. Unfortunately, the model isn't stable for the C96 case, and I have no fix for that yet. Please see that PR for details.

**Comment by guoqing-noaa** [comment]
Close this PR. The issue is addressed at PR https://github.com/ufs-community/ufs-weather-model/pull/2284

**Pull request closed without merging**